patches/minecraft/net/minecraft/client/server/LanServerPinger.java.patch

```diff
@@ -1,6 +1,10 @@
 --- a/net/minecraft/client/server/LanServerPinger.java
 +++ b/net/minecraft/client/server/LanServerPinger.java
-@@ -16,7 +_,7 @@
+@@ -12,11 +_,10 @@
+ import net.minecraftforge.api.distmarker.OnlyIn;
+ import org.slf4j.Logger;
+ 
+-@OnlyIn(Dist.CLIENT)
```

Comment:
Using a patch to remove an OnlyIn isn't correct and should instead be done via forge's [SAS file](https://github.com/MinecraftForge/MinecraftForge/blob/1.19.x/src/main/resources/forge.sas)

Reply:
How do I do that? The file appears to have no entries